applicationset/webhook/webhook.go

```diff
@@ -318,6 +410,12 @@ var azuredevopsAllowedPullRequestActions = []string{
 	"git.pullrequest.updated",
 }
 
+var bitbucketServerAllowedPullRequestActions = []string{
```

Comment:
what aboit `pr:declined` and `pr:deleted`

Reply:
@brunocascio thanks for the hint, added the two mentioned event keys.